glpk/bld.bat

```diff
@@ -1,14 +1,7 @@
-cd w%ARCH%
-copy config_VC config.h
-
-nmake /f Makefile_VC

```

Comment:
Sorry, I should have restored this too when rolling back the first commit. Committing.